tests/onnx/test_onnx_model_export.py

```diff
@@ -252,6 +252,29 @@ def test_model_save_load(onnx_model, model_path):
     assert onnx.checker.check_model.called
 
 
+@pytest.mark.parametrize("save_as_external_data", [True, False])
```

Comment:
```suggestion
@pytest.mark.parametrize("save_as_external_data", [True, False])
@pytest.mark.skipif(Version(onnx.__version__) < Version("1.9.0"), reason="...")
```

Can we skip this test if onnx < 1.9.0?

Reply:
+1

Reply:
done! also changed one of the tests below that had the entire test under an `if Version(onnx.__version__) >= Version("1.9.0"):`